tests/DCPS/BuiltInTopicTest/DataReaderListener.cpp

```diff
@@ -29,9 +29,8 @@ void DataReaderListenerImpl::on_data_available(DDS::DataReader_ptr reader)
 
   try {
     ::Messenger::MessageDataReader_var message_dr = ::Messenger::MessageDataReader::_narrow(reader);
-    if (CORBA::is_nil (message_dr.in ())) {
-      ACE_ERROR((LM_ERROR, ACE_TEXT(
-        "(%P|%t) DataReaderListener: read: _narrow failed.\n")));
+    if (CORBA::is_nil(message_dr.in())) {
```

Comment:
```suggestion
    if (!message_dr) {
```

If we're changing style, it's also reasonable to change this.